### ISSUE TITLE
Implement functionality to parse getSession() response data efficiently

### ISSUE THREAD

**Issue by tnt-freskim-veliu** [sidebase/nuxt-auth#350]
### Describe the feature

To simplify the process of obtaining the user object from the getSession() response, I would like to parse the JSON data directly from the request and retrieve the user object from the "data" property. Currently, I am required to store the entire response and extract the user object each time, which is inefficient.

My response json almost all the time is like this:

``
{
  "data": {...user},
  "status_code": 200,
  "message" : "Success",
}
``

**Comment by BracketJohn** [comment]
Any idea how we should implement this? 

One idea: We could give the option to also specify a jaon pointer to the actual session data, so that the right subfield will be extracted

**Comment by tnt-freskim-veliu** [comment]
Yes, that would be a good idea.

**Comment by BracketJohn** [comment]
cc @Danielwinkelmann  btw for more json-pointer use-cases!

**Comment by Danielwinkelmann** [comment]
@BracketJohn I implement the feature in the new PR

**Comment by valh1996** [comment]
Hi @zoey-kaiser, why did you put this completed if it hasn't been merged for 6 months now and review is pending on the PR?

**Comment by phoenix-ru** [comment]
Closed via #592
